api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -238,14 +238,11 @@ private Runtime getOverrideFromListRuntimes(String googleProject) {
       throw new BadRequestException("Runtime cannot be empty for an update request");
     }
 
-    if (runtimeRequest.getRuntime().getGceConfig() == null
-        && runtimeRequest.getRuntime().getDataprocConfig() == null) {
-      throw new BadRequestException("Either a GceConfig or DataprocConfig must be provided");
-    }
-
     if (runtimeRequest.getRuntime().getGceConfig() != null
-        && runtimeRequest.getRuntime().getDataprocConfig() != null) {
-      throw new BadRequestException("Only one of GceConfig or DataprocConfig must be provided");
+        ^ runtimeRequest.getRuntime().getGceWithPdConfig() != null
```

Comment:
Interesting.. I saw a ^ in this code. 
![Screen Shot 2022-01-27 at 10 51 50 PM](https://user-images.githubusercontent.com/6351731/151484313-33c5d0f5-fef5-42f9-b630-2cadc23684f3.png)